docs/how-tos/protocol.rst

```diff
@@ -2,3 +2,536 @@
 
 How to define a new ``Protocol``
 ================================
+
+The **gufe** :ref:`Protocols <protocol>` system is designed as an *extensible point* of the library,
+allowing developers to write their own methods for performing free energy calculations in a form that can take full advantage of the OpenFE ecosystem.
+
+Our recommendation in this how-to is to start simple;
+we will call out areas where you can iterate further on making your ``Protocol`` more sophisticated.
+
+.. image:: ../_static/gufe_protocol_diagram.svg
+    :alt: The ``gufe`` protocol system.
+
+
+Overview: What makes a ``Protocol``
+-----------------------------------
+
+A complete ``Protocol`` implementation requires several interconnected components:
+
+1. **Settings class**: A Pydantic model defining configuration parameters
```

Comment:
```suggestion
1. **Settings class**: A `Pydantic <https://docs.pydantic.dev/latest/>`_ model defining configuration parameters
```

Reply:
a user probably is familiar with pydantic, but I think adding a link on first mention of an external library is good practice.